frontend/packages/schema-editor/src/components/SchemaInspector/SchemaInspector.tsx

```diff
@@ -41,7 +53,9 @@ export const SchemaInspector = () => {
           <ItemFieldsTab selectedItem={selectedItem} />
         </Tabs.Content>
       ) : (
-        <Alert severity='info'>{t('app_data_modelling.fields_information')}</Alert>
+        <Tabs.Content value={t('schema_editor.fields')}>
+          <Alert severity='info'>{t('app_data_modelling.fields_information')}</Alert>
+        </Tabs.Content>
```

Comment:
Fint at du fikk plassert denne i en `<Tabs.Content>`! Kanskje du også kan flytte inn hele `shouldDisplayFieldsTab`-sjekken, siden det ikke er noen forskjell på de to `<Tabs.Content>`-ene?